LVT/LVT.Services/BetPresenter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using LVT.LVT.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LVT.LVT.Services
+{
+    public class BetPresenter : IVisualizer<Bet>
+    {
+        public string VisualizeToString(Bet bet, string parentNode)
+        {
+            string result = "[{ v: '" + bet.NodeID + "', f: 'Bet" + "<div style=\"font-style:italic\">" + bet.Title + "</div>'}, " + $"'{parentNode}']";
```

Comment:
If we remove the `f: 'Bet` and put it in another variable this can then be more easily inherited from a base class?

Should the `font-style:italic` be hard-coded or a separate variable?

Reply:
Assume these comments are there for all ...Presenter.cs classes

Reply:
I like the idea of base classes for this, because there's an awful lot of repetition throughout the presenter classes, so defo worth doing (I think).

The presentation (if not the format of the string itself) is flexible, but sadly the way it's currently written, all of it is hard coded (which is really bad, I agree).  Making it optional to have something italic or non-italic doesn't really address that underlying issue, I think (but could of course be done).
Addressing the real issue, making it more flexible to begin with (which is what Kieron mentioned earlier, using more classes that are responsible for different aspects of formatting) might be more effort than we will easily be able to put into this over the next few days - but maybe I'm wrong.

Reply:
Completely agree, we'll work on the base class first, then extract anything else afterwards (if time and if it's necessary).